.github/workflows/approve-issues.yml

```
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)